conans/model/conf.py

```diff
@@ -114,6 +114,17 @@ def dumps(self):
         else:
             return "{}={}".format(self._name, self._value)
 
+    def serialize(self):
+        if self._value is None:
+            _value = "!"  # unset
+        elif self._value_type is list and _ConfVarPlaceHolder in self._value:
+            v = self._value[:]
+            v.remove(_ConfVarPlaceHolder)
```

Comment:
This is indeed a challenge to represent in a serialized format.

Reply:
I mean, this serialization is losing information, can reconstruct profile from it.

Reply:
the profile looks like an input of the `conan create`, and you already know it prior calling the command.
you don't have to obtain it as a result of the `conan create` - I believe profile is immutable and never modified by the command, right?
in this case, I think JSON resut of the conan should describe the output of the command - e.g. package ID and package revision created.

Reply:
yeah, that's the information you already should know prior to calling `conan create`, so might be useless